kube-client/src/client/auth/oidc.rs

```diff
@@ -370,7 +364,7 @@ impl Refresher {
                     AUTHORIZATION,
                     format!(
                         "Basic {}",
-                        base64::encode(format!(
+                        BASE64_ENGINE.encode(format!(
```

Comment:
```suggestion
                        base64::engine::general_purpose::STANDARD.encode(format!(
```

This PR accidentally changed this from base64 to base64url.

Reply:
Might want to alias-import `STANDARD as STANDARD_BASE64_ENGINE` or such.

Reply:
Good spot. Thanks.